Gemfile

```diff
@@ -17,7 +17,7 @@ group :system_tests do
 end
 
 group :release do
-  gem 'github_changelog_generator',  :require => false, :git => 'https://github.com/voxpupuli/github-changelog-generator', :branch => 'voxpupuli_essential_fixes'
+  gem 'github_changelog_generator',  :require => false, :git => 'https://github.com/voxpupuli/github-changelog-generator'
```

Comment:
hi. why don't you want to use the bugfix branch?

Reply:
I was receiving an issue early today its seems to have gone now will remove this